zproject/urls.py

```diff
@@ -723,6 +724,12 @@
     path("email_mirror_message", email_mirror_message),
 ]
 
+#  This view accepts a JWT containing an email and returns an API key
+#  and the details for a single user.
+urls += [
+    path("jwt/fetch_api_key", jwt_fetch_api_key),
+]
```

Comment:
I think this should probably be in the API URL namespace, like `api/v1/jwt/fetch_api_key`, perhaps?

Also, I'd move this after `Python Social Auth` in the file?

Reply:
> Also, I'd move this after Python Social Auth in the file?

Yeah seems better, moved it and put it in  `/api/v1`.

Reply:
for what it's worth, the original namespace was decided by looking at `json/fetch_api_key`
```
# Desktop-specific authentication URLs
urls += [
    rest_path("json/fetch_api_key", POST=json_fetch_api_key),
]
```